.github/workflows/linter.yml

```diff
@@ -107,7 +107,10 @@ jobs:
       # Run all tests, including doc tests.
       - name: Run cargo test
         run: |
+          TEMPLATE_DIR=`pwd`
           mv $PROJECT_NAME ${{ runner.temp }}/
+          sed -i "s/{version = \"7.0\", default-features = false}/{path = \"${TEMPLATE_DIR//\//\\\/}\/concordium-std\", default-features = false}/g" ${{ runner.temp }}/$PROJECT_NAME/Cargo.toml
+          sed -i "s/{version = \"4.0\", default-features = false}/{path = \"${TEMPLATE_DIR//\//\\\/}\/concordium-cis2\", default-features = false}/g" ${{ runner.temp }}/$PROJECT_NAME/Cargo.toml
```

Comment:
I don't think this is what we want.

Since `cargo concordium init` will take the template from the repository, we must always ensure that the template that exists works with the published versions of the libraries.

I realize that this complicates things, and it's something we should improve #331